fiatconnect-api.md

```diff
@@ -2352,6 +2353,37 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 9.3.2.6. `PIXAccount`
+
+`PIXAccount`  egg. [PIX](https://www.bcb.gov.br/estabilidadefinanceira/pix) is a Brazilian instant payment method created by the Central Bank (BC) in which funds are transferred between accounts in a few seconds, at any time or day. Pix transfers can be carried out from a checking account, savings account or prepaid payment account.
+
+```
+{
+  accountName: `string`,
+  institutionName: `string`,
+  keyType: `PIXKeyTypeEnum`,
+  key: `string`,
+  fiatAccountType: `FiatAccountTypeEnum.BankAccount`
+}
+```
+
+##### 9.3.2.6.1. `PIXKeyTypeEnum`
+
+`keyType` MUST be one of the following values:
+
+```
+[
+  `EMAIL`,
+  `CPF`,
+  `PHONE`,
+  `RANDOM`
+]
+```
+
+Depending on the type of key, the `key` field follows some validation rules. It MUST be an email, [CPF](https://en.wikipedia.org/wiki/CPF_number), mobile number (including area code) or a random (32 alphanumeric characters) key respectively. A random key is generated by bank. The mobile number follows the [Brazilian telephone number standard](https://pt.wikipedia.org/wiki/N%C3%BAmeros_de_telefone_no_Brasil). Example: DD9XXXXXXXX or 55DD9XXXXXXXX
```

Comment:
```suggestion
If `keyType` is `EMAIL`, `key` MUST be a [valid email](https://en.wikipedia.org/wiki/Email_address#Syntax).

If `keyType` is CPF, `key` MUST be a [CPF number](https://en.wikipedia.org/wiki/CPF_number).

If `keyType` is `PHONE`, `key` MUST be an 11-digit [Brazilian mobile phone number](https://en.wikipedia.org/wiki/Telephone_numbers_in_Brazil). Area code MUST be included. The string MUST match the regex `/[0-9]{11}/` (a series of 11 digits, 0 through 9).

Otherwise, if `keyType` is `RANDOM`, `key` MUST be a string of 32 alphanumeric characters. (In this last case, the key represents a random key generated by the user's bank.) 
```
I rewrote this section to make it clear what rules exist for each `keyType`, and replace another non-english reference with an english one

Reply:
I also went on a limb and suggested an exact format for the phone numbers